master/internal/api_experiment.go

```diff
@@ -1576,9 +1576,13 @@ func (a *apiServer) GetModelDef(
 }
 
 func (a *apiServer) MoveExperiment(
-	_ context.Context, req *apiv1.MoveExperimentRequest,
+	ctx context.Context, req *apiv1.MoveExperimentRequest,
```

Comment:
**MoveExperiment**
- MoveExperiment should require access (to edit? to specifically move experiments?) in the Source Project and Destination Project. 

**GetExperiments**
- If someone makes a GetExperimentsRequest with a project ID, we could check for project-read access

Future **GetExperiments** / likely notes for a future ticket:
- If someone makes a GetExperimentsRequest without a project ID, we should filter the experiments based on experiment-read / project-read access
- As a Paginated response, we are going to have some new ?s in how we return the count of access-filtered Experiments, page 2, etc. If access is down to the project level then we could maybe first get all project IDs where they have access.

Reply:
The plan is to tackle ```MoveExperiment``` and ```GetExperiments``` in the next PR which is applying this same interface to experiments if that is okay. 

Those are interesting points for the future. Counting might be especially tough to do in a performant way.